test/io/test_io_image.py

```diff
@@ -28,6 +29,40 @@ def create_random_img8_torch(height: int, width: int, channels: int, device=None
     return (torch.rand(channels, height, width, device=device) * 255).to(torch.uint8)
 
 
+def _download_image(url: str, filename: str = "") -> str:
+    # TODO: move this to testing
+
+    filename = url.split("/")[-1] if len(filename) == 0 else filename
+    # Download
+    bytesio = io.BytesIO(requests.get(url, timeout=60).content)
+    # Save file
+    with open(filename, "wb") as outfile:
+        outfile.write(bytesio.getbuffer())
+
+    return filename
+
+
+@pytest.fixture(scope='class')
+def png_image(tmp_path_factory):
+    url = "https://github.com/kornia/data/raw/main/simba.png"
+    filename = tmp_path_factory.mktemp("data") / "image.png"
```

Comment:
why do you need to create `data` ? is not tmp_path_factory a new path every time the class implementing the fixture ? actually maybe we need to scope at session level

Reply:
from https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-factory-fixture
> The tmp_path_factory is a session-scoped fixture which can be used to create arbitrary temporary directories from any other fixture or test.

the `tmp_path` is a new temp directory for each test itself

Reply:
The idea here is to just download the PNG and JPG image one time per class or session, so it doesn't get heavy/slow